apps/meteor/server/settings/omnichannel.ts

```diff
@@ -412,6 +412,13 @@ export const createOmniSettings = () =>
 			enableQuery: omnichannelEnabledQuery,
 		});
 
+		await this.add('Livechat_transcript_show_system_messages', false, {
```

Comment:
Does this apply to both transcripts? (Email, pdf)

Reply:
Yes.